integration-tests/test/native-eth.spec.ts

```diff
@@ -140,7 +140,7 @@ describe('Native ETH Integration Tests', async () => {
     )
 
     const receipts = await env.waitForXDomainTransaction(
-      env.ovmEth.withdrawTo(l1Bob.address, withdrawAmount),
+      env.ovmEth.withdrawTo(l1Bob.address, withdrawAmount, '0xFFFF', 0),
```

Comment:
I see a good number of these dummy data and gas fields you have had to add in from this change.  Just curious--have you considered adding a function where you can omit the `data` and `gasLimit` and it reroutes with `bytes(""), 0`?  Seems like nice devex for a lot of the use cases where it's unneeded.

One thing could be to use the same function name and overload the arguments.  This would have the advantage of preserving the same methodIDs for the projects who've implemented the old interface.  Has the downside of the ethers being a [bit uglier](https://github.com/ethers-io/ethers.js/issues/407#issuecomment-458329708).

Or maybe there is an argument against this complexity, which could also be the case. :)

Reply:
> Or maybe there is an argument against this complexity, which could also be the case. :)

The argument is that it's more effort than it's worth while the design is still evolving. This would mean adding two new functions to two L1 gateways and one L2 gateway, which are defined by 8 different solidity files. Then that code should also be tested in the 3 unit testing files, and 2 integration tests which are affected. :)

Reply:
Going to resolve this for now, but happy to revisit.